zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -70,7 +68,6 @@ public void configure(final Context context) {
     pluginRepository.load(configuration.getInterceptorPlugins());
 
     context.setFilter(new ElasticsearchRecordFilter(configuration));
-    indexTemplatesCreated = false;
```

Comment:
This is the root-cause of the failures from the 3 tests in `ElasticsearchExporterIT$IndexSettingsTest`